src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -1678,68 +1678,23 @@ public boolean isCoreClass(TypeElement typeElement) {
     }
 
     /**
-     * Adds the annotation types for the given packageElement.
+     * Return a content tree containing the  annotation types for the given Element.
      *
-     * @param packageElement the package to write annotations for.
-     * @param htmltree the documentation tree to which the annotation info will be
+     * @param element an Element
+     * @param lineBreak if true add new line between each member value
+     * @return the documentation tree to which the annotation info will be
      *        added
```

Comment:
The method is `getAnnotationInfo` so it seems strange/confusing to return an object to which the info _will be_ added (future tense.)  I'm surprised it's not a documentation tree containing the annotation info.
Is this a hangover from the old code?

Reply:
Indeed, the method returns the fully rendered content. I'm fixing this in a follow-up commit (along with other issues).